### PR TITLE
Update Ubuntu to 22.04 from 20.04

### PR DESCRIPTION
**Pull request by satyamz**
### What:

Update Ubuntu to 22.04 from 20.04

### Why:

We are migrating Ubuntu base images to 22.04. 

### Testing:
I built the image and it was successful

### CODE REVIEW

**Review by jacekn — APPROVED** [review]
Looks good. Did you test the exporter? We are maintainers and it should be fairly easy to run it on a box with core running and confirm all works as expected.

**Comment by satyamz** [comment]
Yes. I have a setup ready to test this. Will merge it once I am satisfied. Thanks for the review.

**Comment by satyamz** [comment]
Verified this.

**Pull request merged**
